### ISSUE TITLE
[TestFailure] System.Net.Security.Tests.CertificateValidationRemoteServer.DefaultConnect_EndToEnd_Ok

### ISSUE THREAD

**Issue by rzikm** [dotnet/runtime#96689]
## Build Information
Build: https://dev.azure.com/dnceng-public/public/_build/results?buildId=519054
Build error leg or test failing: System.Net.Security.Tests.CertificateValidationRemoteServer.DefaultConnect_EndToEnd_Ok
Pull request:

## Error Message

Fill the error message using [step by step known issues guidance](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssueJsonStepByStep.md#how-to-create-a-known-issue-step-by-step).

```json
{
  "ErrorMessage": "Authentication failed, see inner exception",
  "ErrorPattern": "",
  "BuildRetry": false,
  "ExcludeConsoleLog": true
}
```
 

### Report
|Build|Definition|Step Name|Console log|Pull Request|
|---|---|---|---|---|
|[572457](https://dev.azure.com/dnceng-public/public/_build/results?buildId=572457)|dotnet/runtime|Build native test components|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/572457/logs/1062)||
|[571721](https://dev.azure.com/dnceng-public/public/_build/results?buildId=571721)|dotnet/runtime|Build and generate native prerequisites|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/571721/logs/285)|dotnet/runtime#98560|
|[570643](https://dev.azure.com/dnceng-public/public/_build/results?buildId=570643)|dotnet/runtime|Build managed product components and packages|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/570643/logs/634)|dotnet/runtime#98560|
|[570266](https://dev.azure.com/dnceng-public/public/_build/results?buildId=570266)|dotnet/runtime|Build product|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/570266/logs/224)|dotnet/runtime#98586|
|[2378595](https://dev.azure.com/dnceng/internal/_build/results?buildId=2378595)|dotnet-runtime|Build managed product components and packages|[Log](https://dev.azure.com/dnceng/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/build/builds/2378595/logs/948)|[#37149](https://dev.azure.com/dnceng/internal/_git/dotnet-runtime/pullrequest/37149)|
|[565220](https://dev.azure.com/dnceng-public/public/_build/results?buildId=565220)|dotnet/runtime|Build managed product components and packages|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/565220/logs/425)|dotnet/runtime#98389|
|[565216](https://dev.azure.com/dnceng-public/public/_build/results?buildId=565216)|dotnet/runtime|Build native test components|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/565216/logs/642)||
|[563894](https://dev.azure.com/dnceng-public/public/_build/results?buildId=563894)|dotnet/runtime|Build native test components|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/563894/logs/816)||
|[558524](https://dev.azure.com/dnceng-public/public/_build/results?buildId=558524)|dotnet/runtime|Run SslStress|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/558524/logs/21)||
|[544248](https://dev.azure.com/dnceng-public/public/_build/results?buildId=544248)|dotnet/runtime|Run SslStress|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/544248/logs/21)||
|[543212](https://dev.azure.com/dnceng-public/public/_build/results?buildId=543212)|dotnet/runtime|Run SslStress|[Log](https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/543212/logs/21)||
#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|1|7|11|

 ### Known issue validation
**Build: :mag_right:** https://dev.azure.com/dnceng-public/public/_build/results?buildId=519054
**Error message validated:** `Authentication failed, see inner exception`
**Result validation:** :x: Known issue did not match with the provided build.
**Validation performed at:** 1/17/2024 9:47:24 AM UTC

**Comment by rzikm** [comment]
Link to test in AzDO: https://dev.azure.com/dnceng-public/public/_build/results?buildId=519054&view=ms.vss-test-web.build-test-results-tab&runId=12192474&resultId=176645&paneView=debug

**Comment by rzikm** [comment]
Triage/reporting blocked by https://github.com/dotnet/runtime/issues/96035

**Comment by rzikm** [comment]
This issue was filed for issues occuring on windows, but I don't see any recent hits on that OS. So this is no longer actionable. I will close this and file a new issue if it resurfaces
